coordinator-work.md

```diff
@@ -0,0 +1,58 @@
+
+While DataRefuge/DataRescue strives to be a distributed effort, there is need for a certain level of coordination to make sure there is no effort duplication between events. We are not going to cover all the tasks that Coordinators tackle in this document; instead we will focus on specific tasks that will help all project participants better understand the overall workflow.
+
+# EDGI Coordinators
+[Add a paragraph on EDGI and links to other relevant documents/sites --> Could someone from EDGI please do that?]
```

Comment:
@titaniumbones?

"EDGI 

The Environmental Data & Governance Initiative (EDGI) has created both an [Event Toolkit](https://envirodatagov.org/event-toolkit/) and [series of tools](https://github.com/edgi-govdata-archiving) to facilitate the preservation of environmental data endangered by the incoming US anti-science administration. We work in a [collaborative and open-source manner](https://github.com/edgi-govdata-archiving/overview/projects/2) to design and improve both the toolkit and tools.